doc/api/schemas/DataRepository.yaml

```diff
@@ -8,6 +8,8 @@ properties:
         type: string
     uri:
         type: string
+    commitHash:
```

Comment:
It is not yet entirely clear how data will be version controlled. `commitHash` sounds very git-like, maybe we should use a more generic term like `dataVersion` or `dataIdentifier`

Reply:
rename to dataIdentifier